modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -23,6 +23,21 @@ You can read more about the Infinite Scale beta phase on https://owncloud.com
 - https://owncloud.com/infinite-scale-beta[Infinite Scale Beta Page]
 - https://owncloud.com/news/infinite-scale-beta/[Infinite Scale Beta is coming]
 
+=== Beta 7
+
+Infinite Scale 2.0.0 beta7 includes stability, security and performance improvements. Web Office does now respect the user's language setting and system administrators can recover a lost admin password.
+
+The most prominent changes in **Infinite Scale 2.0.0 beta7** and ownCloud Web 5.7.0 comprise:
+
+* Web Office applications (like ONLYOFFICE or Collabora Online) will now obey the language setting of the ownCloud Web interface. https://github.com/owncloud/ocis/pull/4399[ocis#4399] https://github.com/cs3org/reva/pull/3156[cs3org/reva#3156] https://github.com/owncloud/ocis/issues/4367[ocis#4367]
+* It is now possible to reset a lost administrator password using the command `ocis idm resetpassword`. https://github.com/owncloud/ocis/issues/4084[ocis#4084] https://github.com/owncloud/ocis/pull/4365[ocis#4365]
+* Encryption and authentication settings have been added to the mail server configuration. https://github.com/owncloud/ocis/pull/4443[ocis#4443]
+* The authentication middleware has been rewritten to increase security and stability. https://github.com/owncloud/ocis/pull/4374[ocis#4374]
+* There's a new endpoint `/app/open-with-web` which allows native clients to open applications like ONLYOFFICE or Collabora Online directly in the web browser. https://github.com/owncloud/ocis/pull/4376[ocis#4367] https://github.com/cs3org/reva/pull/3143[cs3org/reva#3143]
+* Infinite Scale 'extensions' have been renamed to 'services' to better fit their actual job. https://github.com/owncloud/ocis/pull/4407[ocis#4407]
```

Comment:
That was already done for beta1. There were some leftovers that are fixed in this beta.

Reply:
Yes, I know. But we didn't really mention it so far.. should I just remove it?

Reply:
removed